rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -92,3 +92,18 @@ spec:
             alert_team_handle: <!subteam^S04PYECHCCU>
             team: pipelines
             runbook_url: TBD
+        - alert: ResolverControllerDeadlock
+          expr: |
+            sum by (source_cluster) (increase(pending_resolutionrequest_count[2m])) > 0
+          for: 75m
+          labels:
+            severity: critical
+            slo: "true"
+          annotations:
+            summary: >-
+              Tekton resolver controller appears to have stopped processing active resolutionrequests which have not been started yet.
+            description: >-
+              Tekton resolver controller on cluster {{ $labels.source_cluster }} has appeared deadlocked on {{ $value }} resolutionrequests.
+            alert_team_handle: <!subteam^S04PYECHCCU>
+            team: pipelines
+            runbook_url: TBD
```

Comment:
LGTM. Just don't forget to return and add an SOP!

Reply:
the SOP is getting worked over in gitlab and MR 112711 @amisstea  .... if you mean something else though, sorry I am missing it .... if I a mssing it, please clarify - thanks.

Reply:
That's perfect. Ty!

Reply:
likewise thanks